distributed/deploy/ssh.py

```diff
@@ -70,6 +72,7 @@ def __init__(
         connect_options: dict,
         kwargs: dict,
         worker_module="distributed.cli.dask_worker",
+        remote_python=None,
```

Comment:
What do you think about making the default value here `sys.executable`?

```suggestion
        remote_python=sys.executable,
```

I have mixed feelings about this. It would simplify code a little, and also help users understand the expectations for this value because it would be in the docstring. However, on the other hand it would be hard coded in the docs as wherever the python path was in the machine that generated the docs.

Reply:
I don't like the idea that this can be hard coded in the docs. I prefer to keep it None rather than possibly confusing the reader.